### ISSUE TITLE
Incorrect MIB importing

### ISSUE THREAD

**Issue by missilerider** [markabrahams/node-net-snmp#122]
I've experienced that once loaded a MIB file (specifically Cisco Meraki MIB file) the module is imported as 'undefined' instead of module name ('MERAKI-CLOUD-CONTROLLER-MIB').

The issue happens due to the first lines that are as follows:
```
[...]
MERAKI-CLOUD-CONTROLLER-MIB

DEFINITIONS ::= BEGIN
[...]
```
If I join the lines as they should be the name is loaded correctly. The file always loaded correctly anyway.

You need Meraki Dashboard access to download the file (I don't think I can distribute it on my own), although the issue is pretty easy to verify with any other MIB file.

Load process as easy as it can be:
```javascript
    var store = snmp.createModuleStore ();
    store.loadFromFile (mibFile);
    console.dir(Object.keys(store.parser.Modules));
```

Thanks in advance and congratulations for this awesome library!

Alfonso Vila

**Comment by markabrahams** [comment]
Hi @missilerider - I tried separating module name and the DEFINITIONS clause in both the IANAifType-MIB and IF-MIB, like so (for IF-MIB):
```
IF-MIB

DEFINITIONS ::= BEGIN
```
and used your supplied code, but couldn't reproduce the problem.  Output is correct for me:
```
0:'RFC1155-SMI'
1:'RFC1158-MIB'
2:'RFC-1212'
3:'RFC1213-MIB'
4:'SNMPv2-SMI'
5:'SNMPv2-CONF'
6:'SNMPv2-TC'
7:'SNMPv2-MIB'
8:'IANAifType-MIB'
9:'IF-MIB'
```
Can you please upload a MIB file that demonstrates the problem?  Thanks.

**Comment by missilerider** [comment]
Hi Mark. I've found that the MIB download link is public, so here you are:
https://n146.meraki.com/resources/MERAKI-CLOUD-CONTROLLER-MIB.mib
Hope it helps.
Best!

**Comment by markabrahams** [comment]
Thank you @missilerider.  The issue was not handling the leading comment.  Fixed in master now and fix published in version 2.9.2 of the npm.
